include/net-snmp/library/snmp_logging.h

```diff
@@ -123,14 +123,18 @@ void netsnmp_disable_this_loghandler( netsnmp_log_handler *logh );
 NETSNMP_IMPORT
 void netsnmp_logging_restart(void);
 
+#ifndef NETSNMP_FEATURE_REMOVE_LOGGING_STDIO
 NETSNMP_IMPORT
 netsnmp_log_handler *
-netsnmp_create_stdio_loghandler(int is_stdout, int priority, int priority_max,
+netsnmp_register_stdio_loghandler(int is_stdout, int priority, int priority_max,
```

Comment:
Why to declare `netsnmp_register_stdio_loghandler()` in a public header file instead of declaring it static in snmplib/snmp_logging.c?

Reply:
Moved them into the C-file.